Sources/SwiftDocC/Benchmark/Benchmark.swift

```diff
@@ -51,7 +49,7 @@ public class Benchmark: Encodable {
     #endif
 
     /// The list of metrics included in this benchmark.
-    public var metrics: [BenchmarkMetric] = []
+    public fileprivate(set) var metrics: [BenchmarkMetric] = []
```

Comment:
I think keeping the setter `public` would help with flexibility here, for example for clients that want to manually process this array before encoding.